backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/document/model/dsl/impl/ElasticsearchIndexSchemaElementImpl.java

```diff
@@ -76,4 +83,13 @@ public ElasticsearchIndexSchemaObjectField objectField(String relativeFieldName,
 				} );
 	}
 
+	private static void checkRelativeFieldName(String relativeFieldName) {
+		if ( relativeFieldName == null || relativeFieldName.isEmpty() ) {
```

Comment:
Minor nitpick but if we want the code to be consistent after the merge, we should probably inject Search 5 `StringHelper` and use it.

Reply:
Done